_printf.c

```diff
@@ -0,0 +1,40 @@
+#include "main.h"
+
+/**
+  * _printf - function that generatess output in accordance to a given format
+  * @format: format (char, string, int, decimal)
+  * Return: size the output text;
+  */
+int _printf(const char *format, ...)
+{
+	unsigned int i, r_value = 0;
+	int (*format_char)(va_list arg);
+	va_list args;
+
+	if (format == NULL)
+		return (-1);
+
+	va_start(args, format);
+
+	for (i = 0; format[i] != 0; i++)
+	{
+		if (format[i] != '%')
+			_putchar(format[i]);
+		else if (format[i + 1] == '%')
```

Comment:
I think here the condition should be `else if (format[i] == '%' && format[i + 1] == '%')` this confirms that the sequence of characters is of this nature `%%`

Reply:
they are both the same thing. it still works. 
an example is i have this case
printf("I am %c boy %% okay"); so you think the result will be wrong because of my statement right?

Reply:
What of a case where you have something like this `printf("I am c%c");`

Reply:
good still same case
the if statement i wrote checks for if c is % NOT if % is %
thats the need for the i+1